html/elements/textarea.json

```diff
@@ -0,0 +1,869 @@
+{
+  "html": {
+    "elements": {
+      "textarea": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/textarea",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true,
+              "notes": [
+                "Before Firefox 6.0 , when a <code>&lt;textarea&gt;</code> was focused, the insertion point was placed at the end of the text by default. Other major browsers place the insertion point at the beginning of the text.",
```

Comment:
Nit: "Before Firefox 6," (no space between 6 and comma, no leading zero)

Reply:
Fixed (twice)